terraform/env/dev/terraform.tfvars.example

```diff
@@ -0,0 +1,3 @@
+aws_region            = "eu-central-1"
+aws_access_key_id     = ""
```

Comment:
don't use the access key and secret key. For local tests, you can use the profile via export AWS_PROFILE 
In CI/CD we will use this:
https://www.automat-it.com/post/using-github-actions-with-aws-iam-roles

Reply:
> don't use the access key and secret key. For local tests, you can use the profile via export AWS_PROFILE In CI/CD we will use this: https://www.automat-it.com/post/using-github-actions-with-aws-iam-roles

@vanelin I said it to you :relieved:!